src/pages/Aliquot.tsx

```diff
@@ -0,0 +1,279 @@
+                Your source tubes have been marked as discarded.
+              </Success>
+              <Table>
+                <TableHead>
+                  <tr>
+                    <TableHeader>Source Labware</TableHeader>
+                    <TableHeader>Destination Labware</TableHeader>
+                    <TableHeader />
+                  </tr>
+                </TableHead>
+                <TableBody>
+                  {aliquoteTableData.map((data) => (
+                    <tr key={data.destinationLabware?.barcode}>
+                      <TableCell>{data.sourceLabware?.barcode}</TableCell>
+                      <TableCell>{data.destinationLabware?.barcode}</TableCell>
+                      <TableCell>
+                        {data.destinationLabware && (
+                          <LabelPrinterButton
+                            labelsPerBarcode={1}
+                            labwares={[data.destinationLabware]}
+                            selectedPrinter={currentPrinter}
+                            onPrint={handleOnPrint}
+                            onPrintError={handleOnPrintError}
+                          />
+                        )}
+                      </TableCell>
+                    </tr>
+                  ))}
+                </TableBody>
+              </Table>
+
+              <div
+                className="flex flex-row items-center sm:justify-end"
+                data-testid={"printDiv"}
+              >
+                <div className="sm:max-w-xl w-full border-gray-200 p-4 rounded-md bg-gray-100 shadow space-y-2">
+                  {printResult && <PrintResult result={printResult} />}
+                  <LabelPrinter
+                    labelsPerBarcode={1}
+                    showNotifications={false}
+                    labwares={aliquotResult?.aliquot?.labware ?? []}
+                    onPrint={handleOnPrint}
+                    onPrintError={handleOnPrintError}
+                    onPrinterChange={handleOnPrinterChange}
+                  />
+                  <MutedText className="text-right">
+                    Note: 1 label for each destination labware will be printed.
+                  </MutedText>
+                </div>
+              </div>
+            </motion.div>
+          )}
+        </div>
+      </AppShell.Main>
+
+      {showGrayPanel && (
+        <div className="flex-shrink-0 max-w-screen-xl mx-auto">
+          {serverErrors && <Warning error={serverErrors} />}
+
+          <div className="my-4 mx-4 sm:mx-auto p-4 rounded-md bg-gray-100">
+            <p className="my-3 text-gray-800 text-sm leading-normal">
+              Once <span className="font-bold text-gray-900">source tube</span>{" "}
+              is scanned and
+              <span className="font-bold text-gray-900">
+                {" "}
+                number of destination tubes{" "}
+              </span>{" "}
+              are entered, <p>click Aliquot to create destination labwares.</p>
```

Comment:
"labwares" isn't a word. "tubes" would be fine here, since we know what specific type the destinations are.